### PR TITLE
Added passport-beam

### PR DESCRIPTION
**Pull request by dsluijk**
Hello,

This is my first contribution on dt. I write everything, but I'm not sure how to run tests (but i wrote them). I hope this is still good.
- AtlasDev

case 1. Add a new type definition.
- [ ] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

### CODE REVIEW

**Comment by mhegazy** [comment]
Please rename the main file to `index.d.ts` and add a `tsconfig.json` a la https://github.com/DefinitelyTyped/DefinitelyTyped/blob/types-2.0/passport/tsconfig.json

File: passport-beam/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Type definitions for passport-beam 1.0.4
+// Project: https://github.com/alfw/passport-beam
+// Definitions by: AtlasDev <https://github.com/AtlasDev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+///<reference path="../passport/passport.d.ts"/>
```
Comment (mhegazy) [inline passport-beam/index.d.ts]: please remove this.

File: passport-beam/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		id: number
+		username: string;
+		provider: string;
+		email: string;
+		_raw: string;
+		_json: any;
+	}
+}
+
+export Strategy;
```
Comment (mhegazy) [inline passport-beam/index.d.ts]: this is invalid syntax.

File: passport-beam/index.d.ts (hunk 1 above)
Comment (AtlasDev) [inline passport-beam/index.d.ts]: This is required. See the guide: http://definitelytyped.org/guides/contributing.html#content

Comment (mhegazy) [inline passport-beam/index.d.ts]: The guide is meant for the master branch. there have been changes on how the files should be authored in the types-2.0 branch.

**Comment by mhegazy** [comment]
thanks!. waiting on CI server.

**Comment by mhegazy** [comment]
some CI server failures. you should be able to get the same errors locally if you run `tsc` in the passport-beam folder in your branch.

**Comment by AtlasDev** [comment]
I saw them, but was not able to fix them yet. I fixed it now, local tests passes. Now it's just a matter waiting for Travis to pass.

**Pull request merged**